client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
+  }
+
+  getActive() {
+    return this.el.classList.contains('is-active');
+  }
+
+  setActive(value) {
+    if (value) {
+      this.el.classList.add('is-active');
+    } else {
+      this.el.classList.remove('is-active');
+    }
+  }
+
+  toggleActive() {
+    this.setActive(!this.getActive());
+  }
+}
+
+export default Drawer;
```

Comment:
At some point we're going to have to decide at what level of granularity we're going to 'componentise' things, and then I think we'll also have to decide whether we want each and every _thing_ that has a JS component aspect to it to be displayed in the pattern library (I would think probably not – perhaps we'd just want to display larger, complete, composed things).

Meantime, I'll remove this component and display everything in the sidebar.